### ISSUE TITLE
Broken link on ONBOARDING.md

### ISSUE THREAD

**Issue by nguyenbanhducA1K51** [DickinsonCollege/FarmData2#691]
Currently in `farmdata2/farmdata2_module
/README.md`  at section `Adding New Sub-Tabs to a FarmData2 Module` has a link to file `ONBOARDING.md`, but this link is not found.

**Comment by IsaacKBoakye** [comment]
I would like to work on this please!

**Comment by braughtg** [comment]
Super!  Thanks for your interest.  Make a PR and tag me when you have it ready!

**Comment by braughtg** [comment]
@IsaacKBoakye This looks like you may be using the GitKit in your course.  If that is the case you should be claiming an issue on the "upstream" repo for your course rather than on the main project repo.

Looks like the upstream for your course is here: https://github.com/NSCC-ITC-Winter2024-ICOM5015-700-MCr/icom5015-fs-2024

**Comment by Vsaluja** [comment]
Hey I would like to work on this issue in this live project

**Comment by braughtg** [comment]
Super! A pull request with a fix will be welcome. See sections 8-13 of the`CONTRIBUTING.md` document for guidelines.

On Mar 9, 2024, at 11:00, Vansh Saluja ***@***.***> wrote:

Hey I would like to work on this issue in this live project

**Comment by Vsaluja** [comment]
I am getting this error while cloning the fork https://prnt.sc/vHMePRfRIR-E any idea ?

**Comment by braughtg** [comment]
Yes. Looks like some files with bad file names got committed to the repo. I have removed them so it should work if you try again.

On Mar 9, 2024, at 11:22, Vansh Saluja ***@***.***> wrote:

I am getting this error while cloning the fork https://prnt.sc/vHMePRfRIR-E any idea ?

**Comment by braughtg** [comment]
My suggestion here would be to change the link to `ONBOARDING.md` to point to the `docs/Technologies.md` file instead.  It seems to have the links to the resources that are referred to in the sentence that currently contains the broken link.

**Comment by Vsaluja** [comment]
> My suggestion here would be to change the link to `ONBOARDING.md` to point to the `docs/Technologies.md` file instead. It seems to have the links to the resources that are referred to in the sentence that currently contains the broken link.

Got it. Working on it.
